lms/static/js/commerce/views/receipt_view.js

```diff
@@ -24,15 +24,37 @@ var edx = edx || {};
             this.render();
         },
 
-        renderReceipt: function (data) {
-            var templateHtml = $("#receipt-tpl").html(),
+        /**
+         * @param product
+         * @returns {bool} True if product requires user to be verified by Software Secure.
+         */
+        requiresVerification: function(product) {
+            function getAttribute(attribute, defaultValue) {
+                var attr = _.findWhere(product.attribute_values, {name: attribute});
+                if (!attr) {
+                    return defaultValue;
+                }
+                return attr.value;
+            }
+
+            return getAttribute('id_verification_required', true);
+        },
+
+        renderReceipt: function(data) {
+            var templateHtml = $('#receipt-tpl').html(),
                 self = this,
                 context = {
                     platformName: this.$el.data('platform-name'),
                     verified: this.$el.data('verified').toLowerCase() === 'true',
                     is_request_in_themed_site: this.$el.data('is-request-in-themed-site').toLowerCase() === 'true'
```

Comment:
@clintonb Better ask @mattdrayer [about this](https://github.com/open-craft/edx-platform/commit/a9b7e4c63dce2e79f309c5de6bdb9df5c099140b#diff-51cca05dc2d297d186cdc94054145f1eR35), but as far as I can tell, `this.$el.data('is-request-in-themed-site')` returns a string (just like other `jQuery.data` "getter" calls). Than, if what was stored in attribute was boolean there's a quirk:

* When it is stored (either via `data-something` attribute in HTML or via `jQuery.data` call) it is serialized. Booleans serialize as "true" or "false"
* When it is extracted by `jQuery.data`, it returns string, "true" or "false.
* The thing is string "false" is still a trueish value if passed directly to JS `bool` function.

So, a check using `=== "true"` is necessary.